amida-tech/saraswati-hedis-results-api
# [SAR-773]: Update HEDIS Info Structure

Also linting fixes

# Related Tickets



- [SAR-773](https://https://amida.atlassian.net/browse/SAR-773)

# How Things Worked (or Didn't) Before This PR

The HEDIS Info collection was hardcoding the id values. Not a good practice and was holding back elastic support

# How Things Work Now (And How to Test)

Now `measureId` is used instead of `_id`.

To test:

1. Clear your `hedis_info` collection in mongoDB
2. `POST http://localhost:4000/api/v1/measures/info` the values from `initialize/hedis-info.json`
3. Run the dashboard, everything should work as normal. No updates to dashboard required



[SAR-773]: https://amida.atlassian.net/browse/SAR-773?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
